src/handlers.js

```diff
@@ -58,6 +58,12 @@ function zoomStart(chart, event, zoomOptions) {
 }
 
 export function mouseDown(chart, event) {
+  const rect = event.target.getBoundingClientRect();
+  const offsetX = event.clientX - rect.left;
+  const offsetY = event.clientY - rect.top;
+  if (chart.legend && _isPointInArea({x: offsetX, y: offsetY}, chart.legend)) {
+    return;
+  }
```

Comment:
```suggestion
  if (chart.legend) {
    const point = getRelativePosition(event, chart);
    if (_isPointInArea({x: offsetX, y: offsetY}, chart.legend)) {
    return;
    }
  } 
```

Reply:
Where does the `offsetX` and `offsetY` come from..?  And where do the `point` go?

Reply:
Sorry, my suggestion was incomplete. Fixed it.

Reply:
Yep, I tested the new one -- look good now, thanks!